src/ui.jl

```diff
@@ -96,6 +97,9 @@ function TerminalMenus.keypress(m::CthulhuMenu, key::UInt32)
     elseif key == UInt32('r')
         m.toggle = :remarks
         return true
+    elseif key == UInt32('f')
```

Comment:
I think it would be more reasonable to change this keybinding to `e` and remove it from the `:edit` toggle. We had a similar discussion when we introduced `r`emark toggle and removed the `r`-binding from the Revise toggle.

Reply:
Sure, done!